CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# Changelog
+
+Notable changes to this project are documented in this file. The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+Breaking changes:
+
+New features:
+
+Bugfixes:
+
+Other improvements:
+
+## [v6.1.1](https://github.com/purescript-web/purescript-web-dom-parser/releases/tag/v6.1.1) - 2019-08-18
+
+Raised upper bound for `purescript-web-dom`
```

Comment:
I’m not able to at the moment (on mobile) but we should try to manually fill in the prior releases — at minimum the major releases but preferably all.

Reply:
Keeping in mind that the initial releases are here:

https://github.com/toastal/purescript-dom-parser

Reply:
Fixed.